internal/testing/releasehelper.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+var doReplace = flag.Bool("replace", false, "add replace directives to root")
+var doDropReplace = flag.Bool("dropreplace", false, "drop replace directives to root")
+
+func cmdCheck(s string) []byte {
+	fmt.Printf(" -> %s\n", s)
+	fields := strings.Fields(s)
+	if len(fields) < 1 {
+		log.Fatal("Expected \"command <arguments>\"")
```

Comment:
`Expected "command <arguments>"`

Reply:
Not sure what you mean?

Reply:
Ah, I see what happened here!

I suggested that you use the Go backtick string quotes, so you wouldn't have to escape the double quotes. But of course Markdown interpreted my backticks. Let me try again:
```
`Expected "command <arguments>"`
```

Reply:
Done, nice catch